fe/fe-core/src/main/java/org/apache/doris/http/action/VariableAction.java

```diff
@@ -17,6 +17,7 @@
 
 package org.apache.doris.http.action;
 
+import com.google.common.collect.Lists;
```

Comment:
Wrong import order, please refer to `http://doris.incubator.apache.org/master/zh-CN/developer-guide/java-format-code.html`

Reply:
Done